examples/purchaseTester/ios/ReactNativeSample.xcodeproj/project.pbxproj

```diff
@@ -438,7 +438,7 @@
 				COPY_PHASE_STRIP = YES;
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
 				GCC_C_LANGUAGE_STANDARD = gnu99;
 				GCC_NO_COMMON_BLOCKS = YES;
 				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
```

Comment:
I assume these were changed due to a new version of Xcode, but there's some stuff that looks like got changed by React Native itself. Did you maybe try at some point with a newer version of React Native and then reverted the version back but these stuff didn't get reverted?